person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age, :id
+
+  def initialize(age, name: 'Unkown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?(age)
+    true if age >= 18
+    false
+  end
+
+  public
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission
+  end
```

Comment:
- [ ] in the method `can_use_services?` your calling a method called `is_of_age?` however the actual method is called `of_age?`